src/WarningView.tsx

```diff
@@ -43,15 +42,15 @@ const WarningView = () => {
                     </Alert>
                     {device && readbackProtection === 'protected' && (
                         <Alert variant="warning" label="">
-                            <div className="d-flex align-items-center flex-wrap readback-protection-warning">
+                            <div className="d-flex align-items-center readback-protection-warning flex-wrap">
                                 Unable to verify compatible firmware because the
                                 selected device has readback protection enabled.
-                                <Button
+                                <button
```

Comment:
Did you check whether the button keeps the same styling?
If I remember right, even custom button used to get the height, padding and some default styling (though I may be wrong).

Reply:
yes :)